pcdsdevices/epics_motor.py

```diff
@@ -162,6 +163,121 @@ class IMS(PCDSMotorBase):
     This is a subclass of `PCDSMotorBase`
     """
     __doc__ += basic_positioner_init
+    # Bit masks to clear errors and flags
+    _bit_flags = {'powerup': {'clear': 36,
+                              'readback':  24},
+                  'stall': {'clear': 40,
+                            'readback': 24},
+                  'error': {'clear': 48,
+                            'readback': 22,
+                            'mask': 0x7f}}
+    # Custom IMS bit fields
+    reinit_command = Component(EpicsSignal, '.RINI')
+    bit_status = Component(EpicsSignalRO, '.MSTA')
+    seq_seln = Component(EpicsSignal, ':SEQ_SELN')
+    error_severity = Component(EpicsSignal, '.SEVR')
+    part_number = Component(EpicsSignalRO, '.PN')
+
+    def stage(self):
+        """
+        State the IMS motor
+
+        This clears all present flags on the motor and reinitializes the motor
+        if we don't register a valid part number
+        """
+        # Check the part number to avoid crashing the IOC
+        if not self.part_number.get():
+            self.reinitialize(wait=True)
+        # Clear any pre-existing flags
+        self.clear_all_flags(wait=True)
+        self.stage()
+
+    def auto_setup(self):
+        """
+        Automated setup of the IMS motor
+
+        If necessarry this command will:
+            * Reinitialize the motor
+            * Clear powerup, stall and error flags
+        """
```

Comment:
Does this RST look correct in the docs? I recall you need an extra space sometimes for the bulleted lists.